shared/src/commonMain/kotlin/dev/johnoreilly/confetti/ConfettiRepository.kt

```diff
@@ -195,14 +195,21 @@ class ConfettiRepository : KoinComponent {
         }
     }
 
-    fun sessionDetails(
+    fun sessionDetailsQuery(
```

Comment:
In this particular future world, all repositories are only defined in terms of query functions returning ApolloCall.